firefox-ios/Client/Coordinators/CredentialAutofillCoordinator.swift

```diff
@@ -112,6 +113,59 @@ class CredentialAutofillCoordinator: BaseCoordinator {
         }
     }
 
+    @MainActor
+    func showSavedLoginAutofill(tabURL: URL, currentRequestId: String) {
+        let viewModel = LoginListViewModel(
+            tabURL: tabURL,
+            loginStorage: profile.logins,
+            logger: logger,
+            onLoginCellTap: { [weak self] login in
+                guard let self = self else { return }
```

Comment:
```suggestion
                guard let self else { return }
```

Reply:
I've suggested some changes here and below to make the lines a bit more concise. These changes were introduced in Swift 5.8. 

You can read about it [here if you're interested](https://www.hackingwithswift.com/swift/5.8/implicit-self-weak-capture)

Reply:
Ah yes, old habits die hard 😅